template/nuxt/nuxt.config.js

```diff
@@ -64,7 +62,12 @@ module.exports = {
   */
   axios: {
     // See https://github.com/nuxt-community/axios-module#options
-  },<% } %>
+  },<% } %><% if (server === 'firebase') { %>
+
+  /*
+  ** Build directory
+  */
+  buildDir: '../functions/.nuxt',<% } %>
```

Comment:
`npm run dev` still doesn't work as expected when any UI framework's custom template is used. Error displayed on the browser console,

> The client-side rendered virtual DOM tree is not matching server-rendered content. This is likely caused by incorrect HTML markup, for example nesting block-level elements inside <p>, or missing <tbody>. Bailing hydration and performing full client-side render.